src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/mapper/zipfilestatus/ZipFileEnvelopeMapper.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.model.mapper.zipfilestatus;
+
+import uk.gov.hmcts.reform.bulkscanprocessor.entity.Envelope;
+import uk.gov.hmcts.reform.bulkscanprocessor.model.out.zipfilestatus.ZipFileEnvelope;
+
+public final class ZipFileEnvelopeMapper {
+
+    private ZipFileEnvelopeMapper() {
+        // util class
+    }
+
+    public static ZipFileEnvelope fromEnvelope(Envelope envelope) {
+        if (envelope == null) {
+            return null;
+        } else {
+            return new ZipFileEnvelope(
+                envelope.getId().toString(),
+                envelope.getContainer(),
+                envelope.getStatus().name()
+            );
+        }
```

Comment:
please don't consider this message at all, just a blank thought: how about not introducing extra indentation due to if/else statement? for such trivial comparison i could suggest a basic ternary

```java
return envelope == null ? null : new Smth(
   ...
);
```